app/models/manageiq/providers/hawkular/inventory/server_operations.rb

```diff
@@ -27,6 +27,7 @@ def generic_operation(name, action_name, default_params = {}, default_extra_data
             def specific_operation(name, action_name, default_params = {})
               define_method(name) do |ref, params = {}|
                 params[:resourcePath] = ref.to_s
+                params[:operationName] = action_name
```

Comment:
how did this work for other operations before you added this line? it looks like a generic place for not only the remove operation.
or it did not work for other ops as well?